### DISCUSSION TITLE
NetAlertX DIscord Webhook setup

### DISCUSSION THREAD

**Discussion by jerg121** [netalertx/NetAlertX#771] [Q&A] [answered]
Hi, I am currently trying to configure discord webhooks for this. When I run a test I am not receiving any alert to discord. The current webhook setup i have for this is below: If someone can please offer guidance
![Screenshot from 2024-08-23 22-33-57](https://github.com/user-attachments/assets/5de55a80-72dd-441a-8f36-1ea83e0e177b)

**Comment by jokob-sk** [comment]
Hi,

Thansk for using NAX! Can you try to change `WEBHOOK_REQUEST_METHOD` to `GET`?

**Reply by jerg121** [reply]
Yes, I have tried with GET and still do not receive a discord webhook, I have checked the webhook address as a test and did receive a notification from the test

**Reply by jokob-sk** [reply]
I just tested and it seems to be working on my end, can you post your app.log after enabling debug and trying again?

**Reply by jokob-sk** [reply]
Please also check these guides: 

https://github.com/jokob-sk/NetAlertX/blob/main/docs/DEBUG_PLUGINS.md
https://github.com/jokob-sk/NetAlertX/blob/main/docs/DEBUG_TIPS.md

**Reply by jokob-sk** [reply, marked as answer]
Also, is your targetURL correct? Mine starts with `https://discord.com/api/webhooks/`

**Reply by jerg121** [reply]
Discord set the URL to https://discordapp.com/api/webhooks , Once I changed it to https://discord.com/api/webhooks/ it worked!

Thanks for the support and this great application!
